### DISCUSSION TITLE
Doesn't work in Win 11

### DISCUSSION THREAD

**Discussion by myminpins** [FelixdelasPozas/TrayWeather#116] [Q&A] [answered]
Do you know if this should run in Win 11?  I tried installing as administrator and running as administrator but it just flashes and disappears.  I really love this app.  Any suggestions?

**Comment by adem4ik** [comment]
I've tried on Windows 11 Pro Insider Preview 22H2 25145.1011 installed on VBox. Didn't notice any problem during installation and run. Mb you need to check Windows updates or wait for further fixes from Microsoft:
![image](https://user-images.githubusercontent.com/4707112/175992415-584f49df-8ac9-4576-8b4b-ff2141ee5fea.png)

**Reply by myminpins** [reply]
![Screen Shot 06-28-22 at 04 31 PM](https://user-images.githubusercontent.com/6044243/176270204-2b74d283-6c30-49ba-ad26-41094a56a55f.JPG)
![Screen Shot 06-28-22 at 04 33 PM](https://user-images.githubusercontent.com/6044243/176270214-c72e16c1-f95a-4a2d-9e67-9c02d6ee6985.JPG)
I'm using Windows 11 Pro and my computer is up to date but I'm not using the Insider Preview version and it just flashes and never shows an icon in the tray and just disappears.  Very frustrating because WeatherEye, the alternative I was using, just discontinued its product and it no longer works  :(  I've had that icon in my tray for years and years and I'd really like it to work. I checked my task manager and it's there during setup then just vanishes  :(

**Comment by FelixdelasPozas** [comment, marked as answer]
Hi @myminpins . 

I haven't personally tried the application in Windows 11, but as @adem4ik says (and other users also reported) it seems to work. Try the [latest version (1.22)](https://github.com/FelixdelasPozas/TrayWeather/releases/tag/1.22.0). If it still doesn't a bit more information will be useful to solve it. Maybe you could add the information in the "Event viewer" (I suppose Win 11 still has that) to see why it fails?

Regards, 
Felix.

**Reply by myminpins** [reply]
You released a new version!!!  ![Screen Shot 06-30-22 at 06 58 AM](https://user-images.githubusercontent.com/6044243/176649998-bbd8f931-fb25-4097-8734-74fee495d123.JPG)  Thank you so much  :)  Yay!!!!!!!!!!!

**Reply by adem4ik** [reply]
@myminpins omg, Winamp. Looks like Windows XP :D

**Reply by myminpins** [reply]
I LOVE Winamp.  Been using it for many, many years.  And I configure my computer to look as close to Win 7 as I can as I cannot stand the "new and improved" Windows.  :)  Winamp still works perfectly -- use it on my phone, too  :)

**Comment by rawhide338-alt** [comment]
9/25/26 and have the latest issue of your program installed on my Windows 11 machine and it's working just fine.  However one thing I cannot get to work consistently is the startup on start of the computer.  I have to physically select it in order for it to propagate into my tray.  It then updates with no issues.  Anyway I can fix this startup issue?  Thanks
